sdcm/cluster.py

```diff
@@ -1872,10 +1874,43 @@ def run_nodetool(self, sub_cmd, args="", options="", ignore_status=False):
         """
         cmd = self._gen_nodetool_cmd(sub_cmd, args, options)
         result = self.remoter.run(cmd, ignore_status=ignore_status)
-        self.log.debug("Command '%s' duration -> %s s", result.command,
-                       result.duration)
+        self.log.debug("Command '%s' duration -> %s s" % (result.command, result.duration))
+
+        if sub_cmd == 'status':
```

Comment:
this shouldn't be here, this is a general method. Think of it why do I need to run this check always? This was intended to run in `nemesis.log_time_elapsed_and_status` decorator after the `print_nodetool_status`

Reply:
fixed